### DISCUSSION TITLE
YouTube saved file name not always correct.

### DISCUSSION THREAD

**Discussion by ghmagnet** [aclap-dev/video-downloadhelper#1143] [Help and Q&A]
### Your question

I'm on linux.
Version: 9.1.0.42 (stable)
target: mozilla
locale en-GB
co-app: v2.0.19

Quick bug report. On certain youtube videos the file is downloading correctly but the file will be saved as "YouTube-www youtube.mp4" so the title is not being picked up correctly.

regards
magnet :)

### Addon & CoApp version and details

version: 9.1.0.42
target: mozilla
channel: stable
lang: en-GB
coapp: {"found":true,"path":"/home/magnet/.local/share/vdhcoapp/vdhcoapp","version":"2.0.19","new_version":false}
license: {"unneeded":true}
platform: x86-64 linux
UA: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:129.0) Gecko/20100101 Firefox/129.0
{
    "http_media_download_strategy": "inbrowser",
    "used_history_button": true,
    "download_directory": "/media/magnet/spare1/youtube",
    "concurrent_downloads_max": 1,
    "show_thumbnail_in_notification": false,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": true,
    "open_count_store": 1,
    "successfull_dl": 53,
    "never_show_successfull_dl_message": true,
    "download_history": {
        "__serializer_tag": "map",
        "__serializer_value": []
    },
    "history_limit_in_days": 0
}

**Comment by ghmagnet** [comment]
If I switch back to using the legacy mode the files are downloaded and saved with their full correct name, so I'm guessing this issue is related to the new interface.

**Reply by GrampaWildWilly** [reply]
Sounds like a duplicate of #1132.
